tally_ho/apps/tally/models/site_info.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from django.contrib.sites.models import Site
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from tally_ho.libs.models.base_model import BaseModel
+
+
+class SiteInfo(BaseModel):
+    site = models.OneToOneField(Site, on_delete=models.PROTECT)
+    user_idle_timeout = models.PositiveIntegerField()
```

Comment:
I think you should set an explicit default for this field.  This is because you are using `models.OneToOneField` paired with a signal that auto-creates a SiteInfo object whenever a Site object is created - which means that if the SiteInfo model does not have sane defaults for its fields (or has fields that don't have defaults but can't be NULL) then bad things can happen.

Reply:
Makes sense